### PR TITLE
Add ohwellwhatever's handle to the map

### PR DESCRIPTION
**Pull request by ohwellwhatever**
Adding SEA to the map!

### CODE REVIEW

**Comment by ohwellwhatever** [comment]
trying again

**Comment by ebenezergraham** [comment]
@ohwellwhatever .The 'H' in your githubhandle written in your json file should be capital.

File: _pins/ohwellwhatever [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle: ohwellwhatever
```
Comment (ebenezergraham) [inline _pins/ohwellwhatever]: githubHandle

Comment (hollenberry) [reply _pins/ohwellwhatever]: 🎉  this is correct @ohwellwhatever ! Also, you'll need to rename your file and add a `.json` extension. If you need any help, please check out the following video: https://vimeo.com/201011288, or reach out again and let us know if you have questions! :tada:

**Comment by brianamarie** [comment]
@ohwellwhatever I made some changes so the tests will pass. Once everything turns green, I'll go ahead and merge this in since it's been a few days and we want to keep the repository clean. 😄 ✨

**Pull request merged**